### DISCUSSION TITLE
Play Screen

### DISCUSSION THREAD

**Discussion by nateyboi06** [ntsc-rs/ntsc-rs#209] [Ideas]
It would be great to have an option to toggle the iconic blue VHS play screen, along with customizable stop and pause screens. Adding the ability to personalize these would be a neat feature. Example below
![image](https://github.com/user-attachments/assets/d77e2e81-d57f-4644-b7eb-2dbc5a1e361a)

**Comment by notdigidani** [comment]
![image](https://github.com/user-attachments/assets/5c339612-539c-4615-b169-1d2fb62f483f)

**Comment by Maincrosift** [comment]
The idea looks sick, would be pretty cool but that look's pretty difficult for valadaptive to code tbh. let's see if this can happen in the future.

**Comment by eisenbruch** [comment]
Yes please!
But seems like this is a dupe of #342 which has this and other amazing ideas!

**Comment by valadaptive** [comment]
I've decided against implementing this, and added [an FAQ entry explaining why](https://ntsc.rs/docs/faq/#osd):

> Many people have requested that I add on-screen display (OSD) emulation to ntsc-rs. This is the part that displays things like play/pause icons, text, timecodes, and such atop the image.
> 
> Firstly, adding an OSD would require adding a way to control what it says. If you want to switch between different icons (play, pause, etc) then you need to implement keyframing. The standalone ntsc-rs app will never have keyframing, since it is not meant to be general-purpose video editing software.
> 
> Secondly, on-screen displays are unique to each model of VCR. The positioning of text, exact shapes of icons, and fonts are all extremely varied. I could choose to hardcode a specific set of icons and fonts, and deal with a constant influx of requests to add more. Or I could add a way to let people add arbitrary graphics (like text and icons) on top of footage, which is already implemented in real video editing software.
> 
> If you want to render arbitrary text and graphics on top of your footage, you should use video editing software. ntsc-rs is available in plugin form and compatible with most editors. I recommend DaVinci Resolve, which is free.
